### PR TITLE
fix(ui): replaced clubname with respective slug

### PR DESCRIPTION
**Pull request by a1pharooster**

### CODE REVIEW

File: api/requirements.txt [hunk 1]
```diff
@@ -21,3 +21,5 @@ pytz==2022.1
 requests==2.28.0
 sqlparse==0.4.2
 toml==0.10.2
+psycopg2==2.8.6
+
```
Comment (Aksh-Bansal-dev) [inline api/requirements.txt:25]: Is this required?

**Review by Aksh-Bansal-dev — CHANGES_REQUESTED** [review]
These changes are not enough, you also need to update `clubName` to slug in `Club.tsx`.

File: api/requirements.txt (hunk 1 above)
Comment (a1pharooster) [reply api/requirements.txt:25]: I needed that for my docker setup

File: ui/src/pages/Club/index.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const Club: React.FC = () => {
     }
   }, [error]);
   const {clubName} = useParams();
-  const club = clubsData?.find(club => club.name === clubName);
+  const club = clubsData?.find(club => club.slug === clubName);
```
Comment (Aksh-Bansal-dev) [inline ui/src/pages/Club/index.tsx:31]: Change `clubName` to `slug`.

File: api/requirements.txt (hunk 1 above)
Comment (Aksh-Bansal-dev) [reply api/requirements.txt:25]: I don't think this is required if you follow the docs.

**Review by Aksh-Bansal-dev — CHANGES_REQUESTED** [review]

File: api/requirements.txt [hunk 3]
```diff
@@ -21,3 +21,4 @@ pytz==2022.1
 requests==2.28.0
 sqlparse==0.4.2
 toml==0.10.2
+
```
Comment (Aksh-Bansal-dev) [inline api/requirements.txt:24]: Remove this line as well.

**Review by Aksh-Bansal-dev — CHANGES_REQUESTED** [review]
Please follow commit message guidelines as mentioned in the [contributing.md](https://github.com/BitByte-TPC/gymkhana/blob/master/CONTRIBUTING.md#standard-commit-messages).

**Comment by Aksh-Bansal-dev** [comment]
A valid commit message `fix(ui): use slug instead of clubname`.

**Pull request closed without merging**